### ISSUE TITLE
column ordering in admin UI  - ASC is not working, 

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#4661]
## Bug Description
When you click on the column name, your data should be ordered.  Only DESC works  (I can see changing query in SQL debug).
But the next click doesn't change it to ASC. 

## How to Reproduce
 Try to order users or news

 
### PHP Version
7.4
 
## Additional Information
Tested with 2.3.1 and latest git

Edit:
This is first HTML markup
`<a title="Sort" `

After first click:
`<a class="sorted-" title="Sort"`

there should be sorted-asc or sorted-desc... so desc is not working, it is default sorting.

**Comment by Jimmi08** [comment]
This is the reason:
form_handler.php in thead method
```
Array
(
    [field] => [FIELD]
    [asc] => [ASC]
    [from] => [FROM]
)
Array
(
    [field] => news_title
    [amp;asc] => desc
    [amp;from] => 0
    [asc] => 
    [from] => 
)
```
$querypattern :
`field=[FIELD]&asc=[ASC]&from=[FROM]`
after using $tp->filter()
`field=[FIELD]&amp;asc=[ASC]&amp;from=[FROM]`

@Deltik

**Comment by Deltik** [comment]
Regression caused by https://github.com/e107inc/e107/commit/19f48180dce79cbcd5c6a22f254e98e3d83104dd

Bisect log:
```
www-data@e107dev:~/html/e107$ git bisect log
git bisect start
# bad: [404425caf12aa466bb2635dbf63441c2b077afe5] Added PHP 8.1 image to workflow.
git bisect bad 404425caf12aa466bb2635dbf63441c2b077afe5
# good: [d4d0a7485802dca40395e8e0fb1d05b8d9066a22] Make sure to use property attribute on og:type
git bisect good d4d0a7485802dca40395e8e0fb1d05b8d9066a22
# bad: [bb4792bff3a15e67aade0a6ce8d1f26475150dd8] Styling fix.
git bisect bad bb4792bff3a15e67aade0a6ce8d1f26475150dd8
# bad: [ccab9b28b0b003d5388ff523d2c0006310f23ce9] fix #4601 bootstrap5 theme mobile button is not working
git bisect bad ccab9b28b0b003d5388ff523d2c0006310f23ce9
# bad: [455551d10f1c2c85b3cefb629c530e16830c956f] Fix for legacy news image paths.
git bisect bad 455551d10f1c2c85b3cefb629c530e16830c956f
# bad: [fd116c177a25369159f5ab63baa0643e1ad795f7] Meta PHP warnings fix.
git bisect bad fd116c177a25369159f5ab63baa0643e1ad795f7
# bad: [2fa9d10c575a5ac764d1c201d106f7bcbd47b774] `e_form`: Handle attribute HTML entities properly for most form elements
git bisect bad 2fa9d10c575a5ac764d1c201d106f7bcbd47b774
# bad: [2080c772c1be34651a880566e0bb336068368181] Simplify `e_form::get_attributes()`
git bisect bad 2080c772c1be34651a880566e0bb336068368181
# bad: [19f48180dce79cbcd5c6a22f254e98e3d83104dd] Restore `htmlspecialchars()` for `e_parse::filter(…, 'str')`
git bisect bad 19f48180dce79cbcd5c6a22f254e98e3d83104dd
# first bad commit: [19f48180dce79cbcd5c6a22f254e98e3d83104dd] Restore `htmlspecialchars()` for `e_parse::filter(…, 'str')`
```

**Comment by Jimmi08** [comment]
Could you later check to sort of user signature field too?  

All fields work, extended too, just this one has sql:
`QRY=SELECT SQL_CALC_FOUND_ROWS u.*,ue.* from e107_user AS u LEFT JOIN e107_user_extended AS ue ON u.user_id = ue.user_extended_id ORDER BY DESC LIMIT 0, 10`
- missing order by
Not sure if this could be caused with data or number of records...  
Thanks
